### PR TITLE
Adding modified and created to Resource and ResourceShape

### PR DESCRIPTION
**Pull request by sebbader**

### CODE REVIEW

**Comment by sebbader** [comment]
Thanks @HaydarAk for updating the PR! Can not approve it myself, @maboeckmann can you have a look?

File: testing/content/RepresentationShape.ttl [hunk 1]
```diff
@@ -49,7 +49,26 @@ shapes:RepresentationShape
...
+		sh:message "<https://raw.githubusercontent.com/IndustrialDataSpace/InformationModel/master/testing/content/RepresentationShape.ttl> (RepresentationShape): An ids:DigitalContent must have at most one ids:created attribute pointing to a valid xsd:dateTimeStamp."@en ; 
+	] ;
+
+	sh:property [
+		a sh:PropertyShape ;
+		sh:path ids:modified ;
+        sh:maxCount 1 ;
+		sh:datatype xsd:dateTimeStamp ;
+		sh:severity sh:Violation ;
+		sh:message "<https://raw.githubusercontent.com/IndustrialDataSpace/InformationModel/master/testing/content/RepresentationShape.ttl> (RepresentationShape): An ids:modified property of an ids:Representation must have valid xsd:dateTimeStamp values as its object."@en ;
```
Comment (maboeckmann) [inline testing/content/RepresentationShape.ttl:69]: Not 100% happy with the error message. One similar to the one above might be better

File: testing/content/DigitalContentShape.ttl [hunk 2]
```diff
@@ -125,7 +125,25 @@ shapes:DigitalContentShape
...
+		sh:message "<https://raw.githubusercontent.com/IndustrialDataSpace/InformationModel/master/testing/content/DigitalContentShape.ttl> (DigitalContentShape): An ids:DigitalContentShape must have at most one ids:created attribute pointing to a valid xsd:dateTimeStamp."@en ; 
+	] ;
 
+	sh:property [
+		a sh:PropertyShape ;
+		sh:path ids:modified ;
+        sh:maxCount 1 ;
+		sh:datatype xsd:dateTimeStamp ;
+		sh:severity sh:Violation ;
+		sh:message "<https://raw.githubusercontent.com/IndustrialDataSpace/InformationModel/master/testing/content/DigitalContentShape.ttl> (DigitalContentShape): An ids:modified property of an ids:DigitalContent must have valid xsd:dateTimeStamp values as its object."@en ;
```
Comment (maboeckmann) [inline testing/content/DigitalContentShape.ttl:144]: Here as well. Rather have a comment similar to the one for ids:created

File: testing/content/RepresentationShape.ttl (hunk 1 above)
Comment (HaydarAk) [reply testing/content/RepresentationShape.ttl]: Resolved in dbed026

File: testing/content/DigitalContentShape.ttl (hunk 2 above)
Comment (HaydarAk) [reply testing/content/DigitalContentShape.ttl]: Resolved in dbed026

**Pull request merged**